components/scream/src/physics/shoc/shoc_calc_shoc_varorcovar.cpp

```diff
@@ -0,0 +1,15 @@
+#include "shoc_calc_shoc_varorcovar_impl.hpp"
+#include "share/scream_types.hpp"
+
+namespace scream {
+namespace shoc {
+
+/*
+ * Explicit instantiation for doing variance or covariance on Reals 
```

Comment:
Because these kinds of files tend to get copy-pasted a lot, we've been encouraging people to eliminate specific text ("for doing variance or covariance on Reals", in this case) and just keep comments generic.